agent/php_user_instrument.h

```diff
@@ -26,6 +27,22 @@ typedef struct nrspecialfn_return_t (*nrspecialfn_t)(
 
 typedef void (*nruserfn_declared_t)(TSRMLS_D);
 
+typedef struct {
+  bool is_set;
+  uint32_t len;
+  union {
+    char *cp; /* cp = char pointer */
+    const char *ccp; /* ccp = const char pointer */
+  } value;
+} zf_metadata_t;
```

Comment:
Mentioned else where but maybe having a way to tell if the `cp` or `ccp` union field is populated would be helpful for robustly releasing the memory for this structure without having to know how it was populated?

Reply:
Is ef8ebf666e0a17b49c8cde41492782371f5dce95 ok?